docker/node/Dockerfile

```diff
@@ -1,9 +1,8 @@
 FROM node:8-slim
 
-# add a non-privileged user for installing and running
-# the application
-RUN groupadd -g 10001 app && \
-    useradd -d /app -g 10001 -G app -M -s /bin/sh -u 10001 app
```

Comment:
The `node` image creates a `node` user (gid 1000) and `node` group (gid 1000). Since this is only run in local development environment, we don't need it to use 10001 id and don't need to create an app user. So I nixed all that.